lotto645.py

```
@@ -163,100 +193,144 @@ def _try_buying(self, headers: dict, data: dict) -> dict:
+            except Exception as e:
+                print(f"[Error] API JSON Parse Failed: {e}")
+                data = {}
+
+            if not data.get("list"):
+                 return {"data": "no winning data (empty list or API fail)"}
+
+            for item in data["list"]:
+                purchased_date = item.get("eltOrdrDt", "-")
+                round_no = item.get("ltEpsdView", "")
+                money = item.get("ltWnAmt", "-")
+                win_result = item.get("ltWnResult", "")
```

Ruff (0.14.10)

Local variable `win_result` is assigned to but never used

Remove assignment to unused variable `win_result`

(F841)